jsk_maps/srv/UpdateMapCenter.srv

```diff
@@ -0,0 +1,5 @@
+float64 center_latitude
```

Comment:
did you consider using dynamicreconfigure? because if center of map must be initialized just after starting jsk_map node, using param might be better.

Reply:
Sounds good idea.